packages/atomic/src/components/search/atomic-search-box-recent-queries/atomic-search-box-recent-queries.ts

```diff
@@ -143,34 +160,38 @@ export class AtomicSearchBoxRecentQueries {
   }
 
   private renderClear(): SearchBoxSuggestionElement {
-    const partialItem = getPartialRecentQueryClearElement(this.bindings.i18n);
+    const partialItem = getPartialRecentQueryClearElement(
+      this.suggestionsBindings.i18n
+    );
 
     return {
       ...partialItem,
-      content: <RecentQueryClear i18n={this.bindings.i18n} />,
+      content: renderRecentQueryClear({i18n: this.suggestionsBindings.i18n}),
       onSelect: () => {
         this.recentQueriesList.clear();
-        this.bindings.triggerSuggestions();
+        this.suggestionsBindings.triggerSuggestions();
       },
     };
   }
 
   private renderItem(value: string): SearchBoxSuggestionElement {
-    const query = this.bindings.searchBoxController.state.value;
-    const partialItem = getPartialRecentQueryElement(value, this.bindings.i18n);
+    const query = this.suggestionsBindings.searchBoxController.state.value;
+    const partialItem = getPartialRecentQueryElement(
+      value,
+      this.suggestionsBindings.i18n
+    );
     return {
       ...partialItem,
-      content: (
-        <RecentQueriesContainer>
-          <RecentQueryIcon icon={this.renderIcon()} />
-          <RecentQueryText query={query} value={value} />
-        </RecentQueriesContainer>
-      ),
+      content: renderRecentQuery({
+        icon: this.renderIcon(),
```

Comment:
Nitpick, but it feels almost silly to have this as a private method; it's used only once and it's quite trivial 😅